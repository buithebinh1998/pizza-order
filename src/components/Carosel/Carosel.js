import React from "react";
import { Carousel } from "react-responsive-carousel";
import "react-responsive-carousel/lib/styles/carousel.min.css";
import Slide2 from "../../assets/img/slider2.jpg";
import Slide3 from "../../assets/img/slider3.jpg";
import Slide4 from "../../assets/img/slider4.jpg";
import Slide5 from "../../assets/img/slider5.jpg";
const carousel = props => {
  return(
    <div className="carousel">
      <Carousel autoPlay showThumbs={false} infiniteLoop={true} showArrows={false} >
        <div>
          <img src={Slide2} alt="" />
        </div>
        <div>
          <img src={Slide3} alt="" />
        </div>
        <div>
          <img src={Slide4} alt="" />
        </div>
        <div>
          <img src={Slide5} alt="" />
        </div>
      </Carousel>
    </div>
    )
};
export default carousel;