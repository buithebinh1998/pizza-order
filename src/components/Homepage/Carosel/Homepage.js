import React, { Component } from 'react'
import Header from '../Header/Header'
import slider1 from '../../assets/img/slider1.jpg';
import slider2 from '../../assets/img/slider2.jpg';
import slider3 from '../../assets/img/slider3.jpg';
import slider4 from '../../assets/img/slider4.jpg';
class Homepage extends Component {
    
    render(){
        return(
            <div>
                <Header/>
                <div class="slideshow-container">

                    <div class="mySlides fade">
                        <div class="numbertext">1 / 4</div>
                        <img src={slider1} style={{width:"100%"}} alt=""/>
                    </div>

                    <div class="mySlides fade">
                        <div class="numbertext">2 / 4</div>
                        <img src={slider2} style={{width:"100%"}} alt=""/>
                    </div>

                    <div class="mySlides fade">
                        <div class="numbertext">3 / 4</div>
                        <img src={slider3} style={{width:"100%"}} alt=""/>
                    </div>

                    <div class="mySlides fade">
                        <div class="numbertext">4 / 4</div>
                        <img src={slider4} style={{width:"100%"}} alt=""/>
                    </div>

                </div>

                <div style={{textAlign:"center"}}>
                <span class="dot"></span> 
                <span class="dot"></span> 
                <span class="dot"></span> 
                </div>
            </div>
        )
    }
}

export default Homepage;