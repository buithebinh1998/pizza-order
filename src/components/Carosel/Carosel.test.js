import { testSnapshots } from "../../utils/test.util";

import Carosel from "./Carosel";

describe("<Carosel / >", () => {
  testSnapshots(Carosel, [
    {
      props: {},
      description: "default render"
    }
  ]);
});