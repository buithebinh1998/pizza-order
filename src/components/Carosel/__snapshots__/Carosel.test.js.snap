// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Carosel / > snapshots default render 1`] = `
<div
  className="carousel"
>
  <div
    className="carousel-root"
    tabIndex={0}
  >
    <div
      className="carousel carousel-slider"
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <button
        aria-label="previous slide / item"
        className="control-arrow control-prev"
        onClick={[Function]}
        type="button"
      />
      <div
        className="slider-wrapper axis-horizontal"
        style={Object {}}
      >
        <ul
          className="slider animated"
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "MozTransform": "translate3d(-100%,0,0)",
              "MozTransitionDuration": "350ms",
              "MsTransform": "translate3d(-100%,0,0)",
              "MsTransitionDuration": "350ms",
              "OTransform": "translate3d(-100%,0,0)",
              "OTransitionDuration": "350ms",
              "WebkitTransform": "translate3d(-100%,0,0)",
              "WebkitTransitionDuration": "350ms",
              "msTransform": "translate3d(-100%,0,0)",
              "msTransitionDuration": "350ms",
              "transform": "translate3d(-100%,0,0)",
              "transitionDuration": "350ms",
            }
          }
        >
          <li
            className="slide"
            onClick={[Function]}
          >
            <div>
              <img
                alt=""
                src="assets"
              />
            </div>
          </li>
          <li
            className="slide selected"
            onClick={[Function]}
          >
            <div>
              <img
                alt=""
                src="assets"
              />
            </div>
          </li>
          <li
            className="slide"
            onClick={[Function]}
          >
            <div>
              <img
                alt=""
                src="assets"
              />
            </div>
          </li>
          <li
            className="slide"
            onClick={[Function]}
          >
            <div>
              <img
                alt=""
                src="assets"
              />
            </div>
          </li>
          <li
            className="slide"
            onClick={[Function]}
          >
            <div>
              <img
                alt=""
                src="assets"
              />
            </div>
          </li>
          <li
            className="slide selected"
            onClick={[Function]}
          >
            <div>
              <img
                alt=""
                src="assets"
              />
            </div>
          </li>
        </ul>
      </div>
      <button
        aria-label="next slide / item"
        className="control-arrow control-next"
        onClick={[Function]}
        type="button"
      />
      <ul
        className="control-dots"
      >
        <li
          aria-label="slide item 1"
          className="dot selected"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
          value={0}
        />
        <li
          aria-label="slide item 2"
          className="dot"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
          value={1}
        />
        <li
          aria-label="slide item 3"
          className="dot"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
          value={2}
        />
        <li
          aria-label="slide item 4"
          className="dot"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
          value={3}
        />
      </ul>
      <p
        className="carousel-status"
      >
        1 of 4
      </p>
    </div>
  </div>
</div>
`;
